widget/embedded/src/hooks/useConfirmSwap/useConfirmSwap.ts

```diff
@@ -108,13 +102,17 @@ export function useConfirmSwap(): ConfirmSwap {
       { blockchains, tokens }
     );
 
-    const confirmSwapWarnings = generateWarnings(initialQuote, currentQuote, {
-      fromToken,
-      toToken,
-      tokens: tokens,
-      selectedWallets,
-      userSlippage,
-    });
+    const confirmSwapWarnings = generateWarnings(
+      initialQuote ?? undefined,
+      currentQuote,
+      {
+        fromToken,
+        toToken,
+        tokens,
+        selectedWallets,
+        userSlippage,
+      }
+    );
```

Comment:
Can you check this scenario? It seems route is not updated after clicking on resetting liquidity sources.

![Recording 2023-11-21 at 11 59 53](https://github.com/rango-exchange/rango-client/assets/99180934/4e41c119-e7c4-4840-b496-4a3c7ea2dfcf)